fileio/ft_write_data.m

```diff
@@ -875,12 +875,97 @@ function ft_write_data(filename, dat, varargin)
     data.label = hdr.label;
     data.sampleinfo = [1 size(dat,2)];
     
-    % convert the raw data structure to Homer format
-    nirs = fieldtrip2homer(data, 'event', evt);
+    % devide data in nirs channels, stimulus channels and auxillary channels
+    seldat  = startsWith(hdr.chantype, 'nirs');
+    selstim = strcmp(hdr.chantype, 'stimulus');
+    selaux  = ~seldat & ~selstim;
+    
+    % create empty SnirfClass
+    snirf = SnirfClass();
+    
+    % collect information for creation of snirf file
+    source_idx=find(contains(data.hdr.opto.optotype, {'transmitter', 'source'}));
+    detector_idx=find(contains(data.hdr.opto.optotype, {'receiver', 'detector'}));
+    tra=data.hdr.opto.tra;
+    all_wavelengths=data.hdr.opto.wavelength(tra(find(tra>0)));
+    split=median(all_wavelengths);
+    WL1.values=all_wavelengths(all_wavelengths<split);
+    WL2.values=all_wavelengths(all_wavelengths>split);
+    WL1.nominal=round(median(WL1.values),-1);
+    WL2.nominal=round(median(WL2.values),-1);
+    num_WL=2;
+    ft_warning('Fieldtrip assumes that nominal wavelengths of %d and %d nm were used. Please adjust manually when this is not correct', WL1.nominal, WL2.nominal)
```

Comment:
can you change this into 

    ft_warning('assuming that the nominal wavelengths are %d and %d nm')

As with the other comment on ODs the "Please adjust" instruction is not informative enough right now. How would one need to adjust the nominal wavelengths? Or should ft_write_data have an key-value option `nominalwavelength`, and similarly an option `datatype` for OD and the other SNIRF types?